src/Layouts/LayersList/LayerListBox.vala

```diff
@@ -52,9 +53,10 @@ public class Akira.Layouts.LayersList.LayerListBox : VirtualizingListBox {
                 row = old_widget as LayerListItem;
             } else {
                 row = new LayerListItem ();
+                row.row_updated.connect (on_row_updated);
```

Comment:
I'm not super happy about this as I was trying to avoid having a signal per layer as much as possible, that's why the hovered and edited signals are coming from the listbox and triggered on click.
I'm not really sure how to avoid this and listen for when the user presses Enter and edits the layer's name.

Reply:
Not sure I understand why this is necessary. There should probably be one editor per layerlistbox that gets listened to, and an active row. For now if this works, it's not a big deal tbh.

Reply:
Yeah, I was having issues in finding a proper approach to having an editor listening to the active row and handling the editing of the name, same way I do for the hover and select signal, which are handled by the listbox itself and not the single row.
I'll try to rework this.